configs/cms-xcache/config.d/50-cms-xcache-authz.cfg

```diff
@@ -10,4 +10,3 @@
 # Allow connections from all hosts
 xrd.allow host *
 sec.protocol  host
```

Comment:
Is there a point to this line if we're overwriting this later? (I notice the same thing in https://github.com/opensciencegrid/xcache/blob/master/configs/stash-cache/config.d/50-stash-cache-authz.cfg#L18 @matyasselmeci )